steps/feature-list-for-release-notes/0.0.1/step.yml

```diff
@@ -0,0 +1,44 @@
+title: Feature list for release notes
+summary: |
+  Creates a list of features gathered from git commits using given pattern.
+description: |
+  Feature descriptions are derived from commit messages. Using given pattern, one can create a list of the featurees based on previous commits. Also a alternative use would be to create links to appropriate features in the backlog system.
+website: https://github.com/maqoo/bitrise-step-feature-list-for-release-notes
+source_code_url: https://github.com/maqoo/bitrise-step-feature-list-for-release-notes
+support_url: https://github.com/maqoo/bitrise-step-feature-list-for-release-notes/issues
+published_at: 2018-06-20T11:09:15.34508427+02:00
+source:
+  git: https://github.com/DroidsOnRoids/bitrise_set_jira_features.git
+  commit: 581f9442ed3e7fbd69786621b7fbe274187b1660
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- artifact-info
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- jira_default_url: null
```

Comment:
Based on the step description i do not see why jira comes here, please update the step description and title.
Also please provide examples in the input descriptions.